node/events-to-ga/README.md

```diff
@@ -0,0 +1,122 @@
+<!-- Name your function -->
+
+# ⚡ Template Function
+
+<!-- Write short function tagline -->
+
+The function will be triggered by configured Appwrite events and report these events to Google Analytics.
+
+## 🧰 Usage
+
+<!-- Copy section for each endpoint -->
+<!-- Document endpoint method and url  -->
+
+### POST /
+
+<!-- Describe the endpoint -->
+
+Gets the event from the header.
+
+**Parameters**
+
+<!-- Document each expected parameter -->
+
+| Name               | Description                    | Location | Type   | Sample Value          |
+| ------------------ | ------------------------------ | -------- | ------ | --------------------- |
+| x-appwrite-user-id | User ID from Appwrite.         | Header   | String | 642...7cd             |
+| x-appwrite-event   | Describes the triggering event | Header   | String | users.65...f19.delete |
+| x-appwrite-trigger | Type of trigger for function   | Header   | String | event                 |
+
+**Response**
+
+<!-- Provide sample body for successful response -->
+
+Sample `200` Response:
+
+```json
+{
+      "ok": true,
+      "message": "event users.653...df19.delete is send to google analytics",
+}
+```
+
+<!-- If relevant, document error responses -->
+
+Sample `503` Response:
+
+```json
+{ "ok": false, "message": "Response status code when posting event to Google Analytics is 503" }
+```
+
+Sample `401` Response:
+
+```json
+{ "ok": false, "message": "Error Posting Event to Google Analytics" }
+```
+## ⚙️ Configuration
+
+<!-- Update values and remove irrelevant settings -->
+
+| Setting           | Value            |
+| ----------------- | ---------------- |
+| Runtime           | Node (18.0)      |
+| Entrypoint        | `src/main.js`    |
+| Build Commands    | `npm install`    |
+| Permissions       | `any`            |

```

Comment:
Since this function should be triggered by events only, the permissions should probably be empty. No user should be executing this function manually.